arch/arm64/Makefile

```diff
@@ -145,9 +145,15 @@ endif
 
 # Default target when executing plain make
 boot		:= arch/arm64/boot
-KBUILD_IMAGE	:= $(boot)/$(KBUILD_TARGET)
 
-all:	$(KBUILD_TARGET)
+ifeq ($(CONFIG_BUILD_ARM64_APPENDED_DTB_IMAGE),y)
+# Warning! Uncompressed kernel + appended DTB + DTBO WILL NOT WORK!
+KBUILD_IMAGE   := $(KBUILD_TARGET)-dtb
+else
+KBUILD_IMAGE   := $(KBUILD_TARGET)
+endif
```

Comment:
The original commit has this inside `ifeq ($(CONFIG_BUILD_ARM64_KERNEL_COMPRESSION_GZIP),y)`.

Reply:
this way it will work with uncompressed kernel without dtbo

Reply:
I dout uncompressed kernel + appended dtb works? Does it?

Reply:
It was working on 4.19